src/Microsoft.ML.OnnxTransform/Microsoft.ML.OnnxTransform.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeInPackage>Microsoft.ML.TensorFlow</IncludeInPackage>
+    <DefineConstants>CORECLR</DefineConstants>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.ImageAnalytics\Microsoft.ML.ImageAnalytics.csproj" />
+    <PackageReference Include="Microsoft.ML.Scoring" Version="1.0.4-dev47509" />
+  </ItemGroup>
+
+
+  <ItemGroup>
+    <Service Include="{B5B31292-0FED-4ACC-9879-329728881CF3}" />
+  </ItemGroup>
```

Comment:
Wonder, what this for?

Reply:
What this for, and is it necessary for project?

---
In reply to: [220281970](https://github.com/dotnet/machinelearning/pull/942#discussion_r220281970) [](ancestors = 220281970,219610821)

Reply:
Removed.

---
In reply to: [220397246](https://github.com/dotnet/machinelearning/pull/942#discussion_r220397246) [](ancestors = 220397246,220281970,219610821)